wpimath/src/main/native/include/frc/util/PIDConstants.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+namespace frc {
+
+/** A class to configure a PIDController with */
+struct PIDConstants {
+ public:
+  // Factor for "proportion" control
+  const double Kp = 0.0;
+  // Factor for "integral" control
+  const double Ki = 0.0;
+  // Factor for "derivative" control
+  const double Kd = 0.0;
```

Comment:
Since these are public members, they should have docs show up in Doxygen/JavaDoc.
```suggestion
  /// Factor for "proportion" control.
  const double Kp = 0.0;
  /// Factor for "integral" control.
  const double Ki = 0.0;
  /// Factor for "derivative" control.
  const double Kd = 0.0;
```